test/test_ruby2ruby.rb

```diff
@@ -379,6 +379,15 @@ def test_match3_asgn
     util_compare inn, out
   end
 
+  def test_safe_attrasgn
+    inn = s(:safe_attrasgn,
+            s(:call, nil, :x),
+            :y=,
+            s(:lit, 1))
+
+    out = "x&.y = 1"
```

Comment:
This test is incomplete. Needs `util_compare` and is emitting warnings because of unused locals. I'll fix on my side.